containers/unit_tests/TestUnorderedMap.hpp

```diff
@@ -480,6 +480,48 @@ TEST(TEST_CATEGORY, UnorderedMap_consistent_size) {
   ASSERT_EQ(2u, m.size());
 }
 
+namespace Impl {
+
+using map_type = Kokkos::UnorderedMap<int, void, Kokkos::DefaultExecutionSpace>;
+
+KOKKOS_FUNCTION
+void test_insert_to_map_copy(map_type const &input_map, const int i) {
+  auto map = input_map;
+  map.insert(i);
+}
+
+struct TestMapCopy {
+  map_type m_map;
+
+  KOKKOS_FUNCTION
+  void operator()(const int i) const { test_insert_to_map_copy(m_map, i); }
+};
+
+}  // namespace Impl
+
+TEST(TEST_CATEGORY, UnorderedMap_shallow_copyable_on_device) {
+  Impl::TestMapCopy test_map_copy;
+
+  Kokkos::parallel_for(1, test_map_copy);
```

Comment:
Use `TEST_EXECSPACE`.  Otherwise we are testing the exact same thing for each backend that is enabled.

Reply:
Changed in [dd1bd50](https://github.com/kokkos/kokkos/pull/6073/commits/dd1bd50f4d5b1d191d28dbed23d36895fcafe05f).